Source/WebKit/Platform/IPC/cocoa/ConnectionCocoa.mm

```diff
@@ -575,12 +575,16 @@ static void clearNoSenderNotifications(mach_port_t port)
 
 bool Connection::kill()
```

Comment:
I think likely, a better way to do this is to not build `Connection::kill()` at all when `USE(EXTENSIONKIT)`. I think it is only called in one place: `AuxiliaryProcessProxy::terminate()` so we don't need the new virtual function at all it seems?

Reply:
That is a good point. I wonder if it would be good to keep the virtual function though, since we are also calling `kill()` from `Connection::enqueueIncomingMessage` to terminate a misbehaving child process. What do you think? In the latest PR I kept the virtual function because of this.